docs/changelog.rst

```diff
@@ -2,6 +2,11 @@
  Doctr Changelog
 =================
 
+Current
+=======
+- Change deploy directory to required argument. This is a backwards incompatible change. Default deploy without arguments should now read ``doctr deploy .`` (:issue:`128`)
```

Comment:
This should be reworded a bit. The default used to be `docs`, but now there is no default. Maybe something like

> There is no longer a default deploy directory. Specify the deploy directory like `doctr deploy .` or `doctr deploy docs`.